tagged.cabal

```diff
@@ -55,7 +55,8 @@ library
   hs-source-dirs: src
   exposed-modules: Data.Tagged
 
-  if impl(ghc >= 8.10)
+  if impl(ghc >= 9.0)
+    -- these flags may abort compilation with GHC-8.10
```

Comment:
It would be helpful to reference a GHC issue number in this comment.

Reply:
There doesn't seem to be an issue, only a fix MR

Reply:
Added a link